presto-main/src/main/java/com/facebook/presto/cost/HistoryBasedPlanStatisticsProvider.java

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
You delete this class in the next commit, so probably no need to add it.

Reply:
Ah, my git skills are bad